pkgs/by-name/oa/oauth2-proxy/package.nix

```diff
@@ -16,6 +15,10 @@ buildGo123Module rec {
     rev = "v${version}";
   };
 
+  patches = [
+    ./fix-tests-go-1.24.diff
```

Comment:
Have you tried submitting this patch to upstream?

Reply:
No and I won't because it is not downwards compatible and upstream hasn't updated their go version, yet. They will notice when they update that and either do the same or change the code to errors.Is() or so.